packages/relay/src/lib/services/metricService/metricService.ts

```diff
@@ -206,22 +203,20 @@ export default class MetricService {
     executionMode,
     transactionId,
     txConstructorName,
-    callerName,
     cost,
     gasUsed,
-    interactingEntity,
```

Comment:
should we remove these properties from `IExecuteQueryEventPayload` as well? what do you think?

Reply:
You're right. Sorry I wasn't clear enough. Let me clarify. What I meant is to remove both `callerName` and `interactingEntity` wherever we use `IExecuteQueryEventPayload`. Like in

https://github.com/hiero-ledger/hiero-json-rpc-relay/blob/ff0dd5b3d0a60a491947f2521fb80bdd4762cb96/packages/relay/src/lib/clients/sdkClient.ts#L386-L397

wdyt?

Reply:
Oh, totally makes sense. Removing them.